tests/example_workflows/pyxaid2/example_1_Si/step3/run_step3.py

```diff
@@ -137,16 +136,15 @@
 step3.run(params)
 
 
-
 """
-I keep it here just to have it handy - need to find out what to do with this
+#I keep it here just to have it handy - need to find out what to do with this
```

Comment:
note, this section is inactive yet - doesn't fit well in the code's (module's) flow. Lets discuss it separately.